travis-ci.sh

```diff
@@ -2,6 +2,12 @@
 
 set -e -x -o pipefail
 
+if [ "${VIBED_SSL:-openssl}" == "botan" ]; then
+    grep -qF "Have_botan" stream/dub.sdl || echo 'versions "Have_botan"' >> stream/dub.sdl
+    # Make botan mandatory (i.e. non-optional)
+    sed -E 's/(dependency "botan".*) optional=true/\1/' -i stream/dub.sdl
+fi
```

Comment:
The Have_botan part shouldn't be necessary if botan is a dependency.
Usually you have to add it to your dub.selections.json to select an optional dependency, so `dub upgrade` and some `jq` command might work, not much better than your sed line though. We definitely need a dub command for that stuff.
Would also make sense to change openssl's `default=true` to `default=false`.
But weirdly enough botan itself lists openssl as dependency, so you can't really get rid of it.
https://github.com/etcimon/botan/blob/32f01e942dc44b8fbbc5eb7fbd285619c57a42af/dub.json#L127